meteor/client/lib/rundownTiming.ts

```diff
@@ -732,6 +769,10 @@ export interface RundownTimingContext {
 	breakIsLastRundown?: boolean
 	/** Was this time context calculated during a high-resolution tick */
 	isLowResolution: boolean
+	/** The absolute start time of the next break, if any. */
+	nextBreakStartTime?: number
```

Comment:
This should not be called `nextBreakStartTime`, since this can be used for things other than breaks. `nextRundownAnchor` maybe?